components/raftstore/src/store/fsm/apply.rs

```diff
@@ -3002,6 +3051,23 @@ where
     }
 }
 
+impl<EK> HeapSize for Msg<EK>
+where
+    EK: KvEngine,
+{
+    #[inline]
+    fn heap_size(&self) -> usize {
+        match self {
+            Msg::Apply { apply, .. } => apply.entries_mem_size as usize,
+            Msg::Registration(r) => r.region.heap_size(),
```

Comment:
Does `pending_request_snapshot_count` need to be counted?

Reply:
it is included in the trace of `Peer`